lib/dry/system/errors.rb

```diff
@@ -58,5 +58,14 @@ def initialize(name)
         super("component identifier #{name.inspect} must be a symbol")
       end
     end
+
+    # Error raised when trying to stop a component that hasn't started yet
+    #
+    # @api public
+    ComponentNotStaredError = Class.new(StandardError) do
+      def initialize(companent_name)
+        super("You can not stop a component that has not been started yet. Please use `start` method to start the component. system.start[:#{companent_name}]")
```

Comment:
I feel like we could tighten this up a bit:

```ruby
"component +#{component_name}+ has not been started"
```

(I'm not sure we need the usage tip at this point)

Reply:
`companent_name` => `component_name`

Reply:
@timriley we can remove the tip, and I will fix the typo as well